src/lib/testing/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+package(default_visibility=["//visibility:public"])
+
+load("//tools:defaults.bzl", "ng_module")
+
+ng_module(
+  name = "testing",
```

Comment:
We should add another item to #13456 to ensure that the release doesn't include `cdk/testing` or `material/testing`

Reply:
Good idea. I will add it to the issue